tests/unit/neptune/new/attributes/atoms/test_float.py

```diff
@@ -60,3 +63,12 @@ def test_get(self):
             var = Float(run, self._random_path())
             var.assign(5)
             self.assertEqual(5, var.fetch())
+
+    def test_float_warnings(self):
+        run = init_run(mode="debug")
+        with pytest.warns(NeptuneUnsupportedValue):
+            run["infinity"] = float("inf")
+            run["neg-infinity"] = float("-inf")
+            run["nan"] = float("nan")
+
+        run.stop()
```

Comment:
Can we assert here that these particular namespaces weren't created?

Reply:
Added! 😄

Reply:
`run.exists("infinity")`? `__dict__` is internal property and I'm not sure that our abstractions handle it properly